dp_cluster_reg/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class UnexpectedHttpCode(Exception):
+    pass
```

Comment:
Should we perhaps integrate with some logging mechanism here? Would that be valuable?

Reply:
currently we are logging console but putting logging will be beneficial as we can put extra messages required for debugging. we can discuss this and will take as part of another PR.

Reply:
Fine to take this in another PR. Getting this in will help us iterate faster.